packages/eslint-config-godaddy-minimal/index.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  extends: [
+    require('./extends')('eslint-config-godaddy-react-flow')
```

Comment:
I don't think this gives us a "minimal" starting point

Reply:
maybe just call this `eslint-config-godaddy-minimal-flow` ?

Reply:
The idea is to be "minimal" in the sense of "permits a wider range of code styles without configuration". Nothing in these rules requires that teams use react or enable flow, but extending this allows teams to use the same style guide whether or not react and flow are in play.